### DISCUSSION TITLE
Rolling file logs ?

### DISCUSSION THREAD

**Discussion by jknight** [vectordotdev/vector#11155] [Q&A] [answered]
I see there's a [file sink](https://vector.dev/docs/reference/configuration/sinks/file/)

What's the recommended approach for rolling logs ? Is there anything built into vector or best to use logrotate ?

**Comment by jszwedko** [comment, marked as answer]
Hi @jknight !

Yeah, `logrotate` is your best bet. We do have an issue exploring adding similar rotation functionality to Vector here: https://github.com/vectordotdev/vector/issues/2173
